var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -438,7 +451,7 @@ def find_optional_library(name, prefix):
             cxxflags = [(xcompiler + flag) for flag in cxxflags]
             if '+cuda' in spec:
                 cxxflags += [
-                    '-x=cu --expt-extended-lambda -arch=%s' % cuda_arch,
+                    '-x=cu --expt-extended-lambda -arch=sm_%s' % cuda_arch,
```

Comment:
I think Todd describes this flag as being handled automatically by the compiler wrappers when using `CudaPackage`.

Reply:
This is a flag for `nvcc` -- I don't think spack has a wrapper for that -- unless that is a recent addition.